File: messages/src/main/java/com/thundashop/core/sendregning/NewRecipient.java

package com.thundashop.core.sendregning;

public class NewRecipient {
    public Integer number;
    public String customerNumber;
    public String name;
    public String email;
    public String organisationNumber;
    public NewInvoiceRecipientContact contact = new NewInvoiceRecipientContact();
    public NewInvoiceRecipientAdress address = new NewInvoiceRecipientAdress();
}

File: html5apps/terminal/events/core/sendregning/NewRecipient.php
<?php
class core_sendregning_NewRecipient {
	/** @var String */
	public $number;

	/** @var String */
	public $customerNumber;

	/** @var String */
	public $name;

	/** @var String */
	public $email;

	/** @var String */
	public $organisationNumber;

	/** @var core_sendregning_NewInvoiceRecipientContact */
	public $contact;

	/** @var core_sendregning_NewInvoiceRecipientAdress */
	public $address;

}
?>